hazelcast/src/main/java/com/hazelcast/cache/impl/AbstractCacheProxyBase.java

```diff
@@ -141,9 +141,6 @@ public void close() {
         }
         loadAllTasks.clear();
```

Comment:
Not related with this PR but there seems to be a problem with the usage of loadAllTasks in the loop here. Don't we need to work on a copy of the loadAllTasks list for iterator safety?